### ISSUE TITLE
Error running from examples

### ISSUE THREAD

**Issue by andreyferriyan** [tokio-rs/tokio-curl#16]
No error compiling the program. Error when running.

`thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: PerformError { error: Error { repr: Custom(Custom { kind: Other, error: Error { description: "Problem with the SSL CA cert (path? access rights?)", code: 77, extra: None } }) } }', /checkout/src/libcore/result.rs:860`

**Comment by alexcrichton** [comment]
Hm this looks like curl is attempting to probe CA path locations but the permissions are off? Do you know if that is true on your system?

**Comment by andreyferriyan** [comment]
Yup, I'm using this example from `tokio-curl` https://docs.rs/tokio-curl/0.1.9/tokio_curl/

```
Cargo.toml
[dependencies]
tokio-curl="0.1.9"
curl="0.4.8"
futures="0.1.14"
tokio-core="0.1.9"
```

Yes. Working in Windows 10 Pro but not in GNU/Linux Lubuntu 16.04

![2017-08-22](https://user-images.githubusercontent.com/5491449/29544041-5beb7ce2-8720-11e7-9426-85caed0272d9.png)

**Comment by alexcrichton** [comment]
Sorry I don't know where this would come from :(

You may want to try digging around in curl's source to see where this error comes from, but short of that I don't otherwise know how to fix this offhand.

**Comment by andreyferriyan** [comment]
It's okay, @alexcrichton , I'll try to find out the problem. I just want to make sure that the bug is not from `tokio-curl`

**Comment by bluss** [comment]
I had this error message (Debian stretch at this time) for a package that was probably using curl through the same path, in my case cargo-outdated's cargo code. (While cargo had no problem, the difference being of course that cargo-outdated was compiled locally).

I fixed it by uninstalling libcurl3-nss, the nss version does not support reading pem certificates at this time (The problem is something like that). The package manager should ensure that an alternate libcurl library remains installed, otherwise se to that.

After package removals, I rebuilt cargo-outdated and it worked fine.
